components/PopupMenu.js

```diff
@@ -14,7 +14,7 @@ const Popup = styled(Box)`
   border-radius: 8px;
   background: white;
   box-shadow: 0px 4px 8px rgba(20, 20, 20, 0.16);
-  z-index: 1000;
+  z-index: 2000;
```

Comment:
Do you remember what this change fixes? Are we sure it's not going to conflict somewhere else?

Reply:
Yes, this one is to make sure the `PopupMenu` appears on top of the `Hero` container. Note that, 

https://github.com/opencollective/opencollective-frontend/blob/3346025d96f695fe70f672e99448ba142d93d3ac/components/collective-page/hero/Hero.js#L110

So we need an `z-index` higher than 1000 for the Popup to appear on top. My opinion is that the PopupMenu should have priority over any other elements. This is used at three places (excluding Navbar); the recurring contributions page (RecurringContributionsPopUp), in the PeriodFilter, and on the TransactionsDownloadCSV. In all these cases the Popup seems to behave normally. 

![image](https://user-images.githubusercontent.com/12435965/168344558-1c648a81-1d3e-4e4f-8e90-961b16f1f945.png)

![image](https://user-images.githubusercontent.com/12435965/168343814-c04fd8db-085d-4aaa-a933-99728516f127.png)

![image](https://user-images.githubusercontent.com/12435965/168343749-98faabdb-854c-4b4e-ae7f-1335def5537e.png)

Reply:
> My opinion is that the PopupMenu should have priority over any other elements.

I don't especially agree with this statement. I initially implemented the z-index with this behavior because the opposite felt strange:

![Peek 2022-05-19 09-51](https://user-images.githubusercontent.com/1556356/169241668-ac68efbb-d434-4576-a8a2-99537c012a2b.gif)

I've asked design what they think about this [on slack](https://opencollective.slack.com/archives/C0YFZ3MHP/p1652946779314399). It's not a strong opinion on my side, so I'd be happy to change my position if you and/or design think it's the right thing to do.

It feels a bit out of scope for this PR though, since the top bar is never fixed I don't see how it's related?

Reply:
Thanks for opening that thread. I agree, I've changed it back to 1000 so that we have what we have before. 👍🏽 The only problem I was trying to fix here was that the popup comes below the `Hero` component,

![image](https://user-images.githubusercontent.com/12435965/169353132-cae14146-5407-4bfd-ac61-83607283ef37.png)

which I solved by adjusting the hero components z-index instead.

Reply:
The z-index of the hero was set after other components. Unless you're really confident that this won't have any side-effects, could we rather solve this problem by setting a higher z-index on the component that you're adding? Like accept a new `zIndex` prop on `PopupMenu` and do something like:

```jsx
TopbarV2 => (
  ...
  <PopupMenu zIndex="2000">
```

:heavy_check_mark: Simple solution
:heavy_check_mark: No risk of side-effects

Reply:
Makes sense. I've made that change. 👍🏽